.circleci/config.yml

```
@@ -25,7 +25,10 @@ workflows:
               only:
                 - /^Version-v(\d+)[.](\d+)[.](\d+)/
       - prep-deps
-      - test-deps
+      - test-deps-audit
+      - test-deps-depcheck
```

This is the syntax error I believe:
```suggestion
      - test-deps-depcheck:
```